curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/670ea2c4f42def724c77e3c7.md

```diff
@@ -0,0 +1,217 @@
+---
+id: 670ea2c4f42def724c77e3c7
+title: Step 38
+challengeType: 0
+dashedName: step-38
+---
+
+# --description--
+
+Now you need to log the value of `entryDropdown.value`, you'll get the currently selected option from the dropdown. Here's how it works:
+
+The `<select>` element contains several `<option>` elements, each with a value (such as "breakfast", "lunch" or "dinner"). The default selection is "breakfast" because it has the selected attribute.
+
+When you log entryDropdown.value, it will return the value of the option currently selected. At first, this will be "breakfast" but as the user interacts with the dropdown, it will log the value of the currently selected option.
```

Comment:
No it won't. The whole point of this is to show that the value in the global state is only captured once.

Reply:
is this the correct description : 

Now you need to log the value of `entryDropdown.value`, you'll get the currently selected option from the dropdown. Here's how it works:

As the `entryDropdown.value` in a static context (outside of an event listener), it will display the value at the time of code execution, which will not update dynamically as the user interacts with the dropdown.

So here it will capture the initial value (in this case, "breakfast"), and it won't update if the user changes the selection later.